cells/lib/ops/mkStandardOCI.nix

```diff
@@ -64,41 +70,85 @@ in
       '';
     };
     operable' =
-      if debug
+      if hasDebug
       then debugOperable
       else operable;
 
-    setupLinks = cell.ops.mkSetup "links" [] ''
-      mkdir -p $out/bin
-      ${runtimeEntryLink}
-      ${debugEntryLink}
-      ${livenessLink}
-      ${readinessLink}
+    inherit (nixpkgs.dockerTools) caCertificates;
+    setupLinks =
+      cell.ops.mkSetup "links" [
+        {
+          regex = "/bin";
+          mode = "0555";
+        }
+      ] ''
+        mkdir -p $out/bin
+        ${runtimeEntryLink}
+        ${debugEntryLink}
+        ${livenessLink}
+        ${readinessLink}
+      '';
+
+    users = cell.ops.mkUser {
+      user = "nobody";
+      group = "nogroup";
+      uid = "65534";
+      gid = "65534";
+      withRoot = true;
+      withHome = true;
+    };
```

Comment:
It might makes sense to add `root` and `app` (default application runner) users as well

Reply:
Root is added by `withRoot`, already.

`nobody` & `nogroup` is the app user signaling that they are not supposed to signify any impersonation, even if they do on the high number `65534`, in practice.

Unless the app has a wired dependency on some additional entries in nss,  `/etc/pass` & `/etc/group`, this is a good baseline.

Should the app have such a "wired" (i.e. presumptive) dependency, it's probably best to make this explicit to the implementor.